<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainPrepare</class>
 <widget class="QWidget" name="TrainPrepare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPlate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="colorlabel">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Define NoData Value (The codes of land use types should be consecutive integers and start from 1, not 0 )</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorgroupBox">
     <property name="title">
      <string>Properties of Land Use Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="ColorTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sureBtn">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
